src/api/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:18.16.0
+
+# Create api directory
+WORKDIR /api
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY . .
+
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --omit=dev
+
+# Bundle app source
+COPY . .
```

Comment:
```suggestion
```
This one can be removed now since we're copying everything on an earlier line.

Reply:
Removed the extra COPY . .